test/data-background-newstyle.adoc

```diff
@@ -0,0 +1,51 @@
+= Test slide deck
+:imagesdir: images
+:backend: revealjs
+
+[%notitle]
+== Opening
+
+image::cover.jpg[background, size=cover]
+
+== canvas
+
+image::cover.jpg[canvas, size=cover]
+
+== !
+
+image::70s.jpg[background, size=cover]
+
+image::meme-2.jpg[width=500px]
+
+== I have no background
+
+== !
+
+image::meme-7.png[]
+
+== !
+
+image::70s.jpg[background,size=contain]
+
+== hey
+
+image::bio.jpg[background,size="100px"]
```

Comment:
Technically, the quotes around an attribute value are only required if the value has commas or spaces. Just FYI.